lib/std/fs/test.zig

```diff
@@ -13,7 +13,8 @@ const File = std.fs.File;
 const tmpDir = testing.tmpDir;
 const tmpIterableDir = testing.tmpIterableDir;
 
-test "Dir.readLink" {
+const readLink = Dir.readLink;
+test readLink {
```

Comment:
I'm not sure if this pattern is the best.
I can see two options that would be better.
1. allow `test Dir.readLink`
2. just put the test right after we define readlink in Dir. This would also allow us to do the same thing for functions on generics where can't just define `const F = ...;`

Reply:
I think option 2 is preferable over all the alternatives.

Reply:
I've left it for now and only done the unambiguous ones. We can come back to the harder ones later, but I didn't want to block on this.